tests/integration/test_sqs.py

```diff
@@ -58,6 +59,7 @@
 
 THIS_FOLDER = os.path.dirname(os.path.realpath(__file__))
 TEST_LAMBDA_ECHO_FILE = os.path.join(THIS_FOLDER, "lambdas", "lambda_echo.py")
+TEST_LAMBDA_TAGS = {"tag1": "value1", "tag2": "value2", "tag3": ""}
```

Comment:
nitpick: Could probably be renamed to `TEST_TAGS`, as these tags do not seem to be related to Lambdas in this case. (not critical, though)..